website/docs/configuration/package-curations.md

```diff
@@ -57,6 +57,7 @@ The structure of the curations file consist of one or more `id` entries:
   curations:
     comment: "An explanation why the curation is needed or the reasoning for a license conclusion"
     purl: "pkg:Maven/com.example.app/example@0.0.1?arch=arm64-v8a#src/main"
+    authors: ["The names of the authors"]
```

Comment:
In YAML, a list needs to be declared as follows:

```
  authors:
  - "Name of one author"
  - "Name of another author"
```

Reply:
done

Reply:
done